### ISSUE TITLE
[Feature]:  Add an accuracy test for super-v3

### ISSUE THREAD

**Issue by suyoggupta** [NVIDIA/TensorRT-LLM#10056]
### 🚀 The feature, motivation and pitch

Before we start diving deep into perf optimization for super-v3, let's add a MMLU / GSM8K test using the pretraining ckpt. example: https://sourcegraph.com/github.com/NVIDIA/TensorRT-LLM/-/blob/tests/integration/defs/accuracy/test_llm_api_autodeploy.py?L155

@nvchenghaoz 

### Before submitting a new issue...

- [x] Make sure you already searched for relevant issues, and checked the [documentation](https://nvidia.github.io/TensorRT-LLM/) and [examples](https://github.com/NVIDIA/TensorRT-LLM/tree/main/examples) for answers to frequently asked questions.

**Comment by suyoggupta** [comment]
@nvchenghaoz : could you please prioritize this? thanks.

cc: @galagam

**Comment by galagam** [comment]
Test is crashing in main ToT. 
Regression within #16b02530 -> #4fc603627 
Bisecting.

```
[12/22/2025-12:09:07] [TRT-LLM] [RANK 1] [E] Failed to initialize executor on rank 1: Error(s) in loading state_dict for GraphModule:
	size mismatch for backbone.layers.1.mixer.experts.64.up_proj.weight: copying a param with shape torch.Size([2688, 1024]) from checkpoint, the shape in current model is torch.Size([2688, 4096]).
	size mismatch for backbone.layers.1.mixer.experts.64.down_proj.weight: copying a param with shape torch.Size([1024, 2688]) from checkpoint, the shape in current model is torch.Size([4096, 2688]).
	size mismatch for backbone.layers.1.mixer.experts.65.up_proj.weight: copying a param with shape torch.Size([2688, 1024]) from checkpoint, the shape in current model is torch.Size([2688, 4096]).
	size mismatch for backbone.layers.1.mixer.experts.65.down_proj.weight: copying a param with shape torch.Size([1024, 2688]) from checkpoint, the shape in current model is torch.Size([4096, 2688]).
	size mismatch for backbone.layers.1.mixer.experts.66.up_proj.weight: copying a param with shape torch.Size([2688, 1024]) from checkpoint, the shape in current model is torch.Size([2688, 4096]).
	size mismatch for backbone.layers.1.mixer.experts.66.down_proj.weight: copying a param with shape torch.Size([1024, 2688]) from checkpoint, the shape in current model is torch.Size([4096, 2688]).
	size mismatch for backbone.layers.1.mixer.experts.67.up_proj.weight: copying a param with shape torch.Size([2688, 1024]) from checkpoint, the shape in current model is torch.Size([2688, 4096]).
```

**Comment by galagam** [comment]
> Test is crashing in main ToT. Regression within #16b02530 -> #4fc603627 Bisecting.
> 
> ```
> [12/22/2025-12:09:07] [TRT-LLM] [RANK 1] [E] Failed to initialize executor on rank 1: Error(s) in loading state_dict for GraphModule:
> 	size mismatch for backbone.layers.1.mixer.experts.64.up_proj.weight: copying a param with shape torch.Size([2688, 1024]) from checkpoint, the shape in current model is torch.Size([2688, 4096]).
> 	size mismatch for backbone.layers.1.mixer.experts.64.down_proj.weight: copying a param with shape torch.Size([1024, 2688]) from checkpoint, the shape in current model is torch.Size([4096, 2688]).
> 	size mismatch for backbone.layers.1.mixer.experts.65.up_proj.weight: copying a param with shape torch.Size([2688, 1024]) from checkpoint, the shape in current model is torch.Size([2688, 4096]).
> 	size mismatch for backbone.layers.1.mixer.experts.65.down_proj.weight: copying a param with shape torch.Size([1024, 2688]) from checkpoint, the shape in current model is torch.Size([4096, 2688]).
> 	size mismatch for backbone.layers.1.mixer.experts.66.up_proj.weight: copying a param with shape torch.Size([2688, 1024]) from checkpoint, the shape in current model is torch.Size([2688, 4096]).
> 	size mismatch for backbone.layers.1.mixer.experts.66.down_proj.weight: copying a param with shape torch.Size([1024, 2688]) from checkpoint, the shape in current model is torch.Size([4096, 2688]).
> 	size mismatch for backbone.layers.1.mixer.experts.67.up_proj.weight: copying a param with shape torch.Size([2688, 1024]) from checkpoint, the shape in current model is torch.Size([2688, 4096]).
> ```

Bisected to https://github.com/NVIDIA/TensorRT-LLM/pull/9751
@2ez4bz for awareness
